apps/developer-hub/src/components/Callout/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { InfoBox, VARIANTS } from "@pythnetwork/component-library/InfoBox";
+import type { ComponentProps, ReactNode } from "react";
+
+import { Case } from "../Case";
+
+type Props = ComponentProps<"div"> & {
+  icon?: ReactNode;
+  header?: ReactNode;
+  variant?: (typeof VARIANTS)[number] | undefined;
+};
+
+const DEFAULT_ICONS: Record<(typeof VARIANTS)[number], string> = {
+  neutral: "⦿",
+  info: "💡",
+  warning: "⚠️",
+  error: "❗",
+  data: "💾",
+  success: "🎉",
```

Comment:
Should we use icons from phosphor-icons to be consistent with the rest of our iconography?

I also think you should add the default icons to the underlying `InfoBox` component directly, it makes sense to me that it would have sane defaults.